### ISSUE TITLE
 scheme is not valid.

### ISSUE THREAD

**Issue by niypoo** [0maru/twitter_login#45]
always show up this warning in debug console,

`[AuthenticationSession] Error: The provided scheme is not valid. A scheme must not include any special characters, such as ":" or "/"`

and the auth process complete success, and I check the scheme url in info.plist it's without any `/` or `:` what is wrong

**Comment by niypoo** [comment]
```
final TwitterLogin twitterLogin = TwitterLogin(
        apiKey: Config.twitterToken,
        apiSecretKey: Config.twitterSecret,
        redirectURI: 'twitter-xxxxxxx://',
      );
```

```

<key>CFBundleURLSchemes</key>
<array>
<string>twitter-xxxxxxx</string>
</array>
```

**Comment by xmkevinchen** [comment]
Hitting the same warning in the debug console

**Comment by 0maru** [comment]
@niypoo 
Sorry it took so long.
This has been fixed in 3.0.8.

**Comment by 0maru** [comment]
@niypoo 
Sorry it took so long.
This problem has been resolved by changing to use only the RedirectURI scheme.
Please update your plugin version to v3.0.8
